app/src/test/java/com/example/marvelapp/domain/usecase/SaveCharacterDataBaseUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.marvelapp.domain.usecase
+
+import com.example.marvelapp.domain.entity.CharacterEntity
+import com.example.marvelapp.domain.local.LocalMarvelDataBase
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import org.junit.Before
+import org.junit.Test
+
+class SaveCharacterDataBaseUseCase {
+    private var localMarvelDataBase: LocalMarvelDataBase = mock()
+    private var characters: List<CharacterEntity> = mock()
+
+    @Before
+    fun setUp() {
+        whenever(localMarvelDataBase.saveCharacters(characters)).then {}
```

Comment:
did you forget the **.thenReturn(...)**

Reply:
Now its no more, i erased that method, because i did not use it. Thx